README.md

```diff
@@ -1,54 +1,104 @@
-# Kwil DB
-
-The database for Web3.
+<div align="center">
+    <h1>Kwil DB</h1>
+    <p><strong>The database for Web3.</strong></p>
+    <img src="https://github.com/kwilteam/docs/blob/main/static/img/kwil%20social-card.jpg" alt="banner" /><br/>
+    <a href="https://github.com/kwilteam/kwil-db/blob/main/LICENSE.md">
+        <img alt="GitHub License" src="https://img.shields.io/github/license/kwilteam/kwil-db">
+    </a>
+     <a href="https://github.com/kwilteam/kwil-db/releases">
+        <img src="https://img.shields.io/github/v/release/kwilteam/kwil-db" alt="Release">
+    </a>
+    <a href="https://github.com/kwilteam/kwil-db/actions">
+        <img src="https://github.com/kwilteam/kwil-db/actions/workflows/ci.yaml/badge.svg" alt="Build Status">
+    </a>
+    <a href="https://github.com/kwilteam/kwil-db/blob/main/go.mod">
+        <img src="https://img.shields.io/github/go-mod/go-version/kwilteam/kwil-db" alt="Go Version">
+    </a>
+    <a href="https://godoc.org/github.com/kwilteam/kwil-db">
+        <img src="https://godoc.org/github.com/kwilteam/kwil-db?status.svg" alt="GoDoc">
+    </a>
+    <a href="https://goreportcard.com/report/github.com/kwilteam/kwil-db">
+        <img src="https://goreportcard.com/badge/github.com/kwilteam/kwil-db" alt="Go Report Card">
+    </a>
+    <a href="https://discord.com/invite/HzRPZ59Kay">
+        <img alt="Discord" src="https://img.shields.io/discord/819855804554543114?logo=discord">
+    </a>
+</div>
```

Comment:
All this HTML is kinda an eye sore.  

The badges format alright with just this:

```markdown
# dcrdata

[![License](https://img.shields.io/github/license/kwilteam/kwil-db)](https://github.com/kwilteam/kwil-db/blob/main/LICENSE.md)
[![Release](https://img.shields.io/github/v/release/kwilteam/kwil-db")](https://github.com/kwilteam/kwil-db/releases)
[![Build Status](https://github.com/kwilteam/kwil-db/actions/workflows/ci.yaml/badge.svg)](https://github.com/kwilteam/kwil-db/actions)
...
```


Also, a 3MB jpg loading from the README is a bit of a faux pas. Two main problems are that it's 5886x3000, which is about 10x larger than needed, and it's saved at 100% quality, which mean's essentially no jpeg compression.

Here it is at 1256x640 with 85% quality:

![kwil social-card-x](https://github.com/kwilteam/kwil-db/assets/140431406/98ccd576-e217-49d9-9d8f-d1bdc8f8d89c)

That looks good for the README and it's only 72.5 KiB, which loads almost instantly.